x-pack/plugins/security_solution/public/actions/filter/generic/filter_in.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * This module does not have any external plugin dependency on purpose
+ * It will be moved to a package in following iterations
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { CellAction } from '@kbn/cell-actions';
+import type { FilterManager } from '@kbn/data-plugin/public';
+import type { Filter } from '@kbn/es-query';
+
+export const FILTER_IN = i18n.translate('xpack.securitySolution.actions.filterIn', {
+  defaultMessage: 'Filter In',
+});
+const ID = 'filterIn';
+const ICON = 'plusInCircle';
+
+export const createFilterInAction = ({
```

Comment:
Hey, yes it is explained in the description:
`FilterIn/Out generic actions extracted to actions/filters/generic directory (they will be moved to their own package in following iterations)`

Reply:
That's why they don't get the `filterManager` from KibanaServices in the generic Filter actions, it is received by parameters to prevent adding the Kibana plugin dependency. These Filter actions can be moved directly to a package and everything will keep working